### ISSUE TITLE
Foundry cheatsheet -> Foundry book

### ISSUE THREAD

**Issue by dabit3** [foundry-rs/book#489]
I spoke with @gakonst about merging the [Foundry Cheatsheet](https://github.com/dabit3/foundry-cheatsheet) into Foundry Book.

I put this repo together to have a more concise one-page overview of some of the things I was using the most, and many people ended up finding it helpful.

But it makes more sense not to have it separate from the rest of the learning material in Foundry Book, as to have less fragmentation.

I wanted to know what you all think of adding this to the Foundry book, and if so where you might think it should go?

Thanksk

**Comment by skaunov** [comment]
Personally I well comfortable with reference guide since there 
one-pagers for `anvil`, `forge`, and `cast`.

But if there will be also a cheatsheet I would use Solidity experience, 
and placed it more or less like they had.

**Comment by Leutwiler** [comment]
In my opinion, we should add it to a new Appendix section. We could even write about the existence of the cheatsheet in the First Steps with Foundry section, so the reader would be knowledgeable about the cheatsheet right at the start of the book.

As you created the cheatsheet, I don't know if you personally want to add it to the Foundry Book, but if you don't mind I can do it for you! Just reply me about it so I can start :grinning:

**Comment by gakonst** [comment]
I'd maybe restructure the book so that the cheatsheet comes first in the intro, and then links out to the other sections to get more granular?

**Comment by dabit3** [comment]
@gakonst can I submit a PR to get this rolling? from there, we can discuss any changes. Just lmk.

**Comment by gakonst** [comment]
Yep go for it - trust your judgment, also wouldn't be afraid of refactoring/reordering things already in the book
